### ISSUE TITLE
/services throws 500 internal server error

### ISSUE THREAD

**Issue by radicalray** [attendanceproject/djattendance#446]
Get this error in the logs:

```
[pid: 19083|app: 0|req: 26/27] 192.168.12.5 () {42 vars in 804 bytes} [Wed Jan  4 19:24:13 2017] GET /services => generated 0 bytes in 1 msecs (HTTP/1.1 301) 2 headers in 115 bytes (2 switches on core 0)
No current week available outside of term
ERROR:django.request:Internal Server Error: /services/
Traceback (most recent call last):
  File "/home/ftta-ap/.virtualenvs/prod/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./services/views.py", line 660, in services_view
    week_start, week_end = cws.week_range
AttributeError: 'NoneType' object has no attribute 'week_range'
```

**Comment by jontey** [comment]
nvm.. I'll take a look at it

**Comment by billyeh** [comment]
Thanks @jontey

**Comment by billyeh** [comment]
Fixed in #507
